lib/Calendar.js

```diff
@@ -89,7 +89,9 @@ function createCalendar(request) {
 
             const xmlRequest = byTimeTemplate({ start, end });
 
-            return request(this.config, "REPORT", 1, xmlRequest).then(xmlParser.parseEvents);
+            return request(this.config, "REPORT", 1, xmlRequest)
+                .then(xmlParser.parseEvents)
+                .then(events => events.filter(event => moment(event.data.start).isSameOrAfter(start, "day")));
```

Comment:
According to the `Scrapegoat` README, this is what the `.getEventsByTime()` method is supposed to do:

> Fetch all events which occur between start and end (have to be valid iCal Dates). If you leave start and end out, you'll get all upcoming events from today. Passing only one date as a parameter returns all upcoming events from that date. 

So, in as much as the CalDAV spec returns all the expanded events even from the past, this method assures anybody who uses it that they will only get back the upcoming events. This is why I filter out all occurrences from the past here.

If we need to fetch all the events, then the `.getAllEvents()` method is the one that returns all the instances of the recurring events.